packages/ontario-frontend-cli/create/questions.js

```diff
@@ -16,22 +16,22 @@ const createQuestions = [
     type: 'input',
     name: 'projectName',
     message:
-      'Specify your project name for the package.json file (lowercase, hyphens, and underscores only):\n',
+      'What is your project named?\n',
```

Comment:
I would like to keep the `(lowercase, hyphens, and underscores only)` despite having the validation for such, to help them succeed the first time.

Thoughts?

Reply:
I'm not bothered one way or the other – this was mostly getting at a simpler way to ask the question (as opposed to introducing `package.json` as something to think about), and I quickly looked to the usual Next/SvelteKit/Gatsby CLIs to see how this question is worded, which is pretty much "What is your project named?".

While I like how succinct that question is, I'm fine as long to add the hint in parentheses, so long as the actual question itself is presented more plainly. Could be something like `What is your project named? (Use only lowercase, hyphens, and underscores only.)`